source/text.cpp

```diff
@@ -664,7 +664,7 @@ spv_result_t SetHeader(uint32_t* words, const uint32_t bound) {
 
   words[SPV_INDEX_MAGIC_NUMBER] = SpvMagicNumber;
   words[SPV_INDEX_VERSION_NUMBER] =
-      SPV_SPIRV_VERSION_WORD(SPV_SPIRV_VERSION_MAJOR, SPV_SPIRV_VERSION_MINOR);
+      spvVersionForTargetEnv(SPV_ENV_UNIVERSAL_1_0_4);
```

Comment:
Might be better to add something to `Version.cpp` or `libspirv.h` that sets the default binary-output version and just reference that here.